spec/features/admin/merchants/index_spec.rb

```diff
@@ -240,6 +237,19 @@
           expect(page).to have_content("available")
         end
       end
+
+      it "has sections for enabled/disbaled merchants" do
+        merchant_1 = Merchant.create!(name: 'Brylan', status: 0)
+        merchant_2 = Merchant.create!(name: 'Antonio', status: 0)
+        merchant_3 = Merchant.create!(name: 'Chris', status: 1)
+        merchant_4 = Merchant.create!(name: 'Craig', status: 1)
+
+        visit "/admin/merchants"
+        expect(page).to have_content("Enabled Merchants")
+        expect(page).to have_content("Disabled Merchants")
+        expect(merchant_4.name).to appear_before(merchant_1.name)
+        expect(merchant_3.name).to appear_before(merchant_2.name)
+      end
```

Comment:
This west is where we could utilize a within block: `within '#enabled-merchants' do`; inside of that block we would expect the page to have only enabled merchants, and to_not have disabled merchants. 

The opposite can be done in a `within '#disabled-merchants' do` block